platform/vs/docker-sonic-vs/start.sh

```diff
@@ -52,21 +54,26 @@ else
     # generate and merge buffers configuration into config file
     if [ -f /usr/share/sonic/hwsku/buffers.json.j2 ]; then
         sonic-cfggen -k $HWSKU -p /usr/share/sonic/device/$PLATFORM/$PLATFORM_CONF -t /usr/share/sonic/hwsku/buffers.json.j2 > /tmp/buffers.json
+        buffers_cmd="-j /tmp/buffers.json"
     fi
     if [ -f /usr/share/sonic/hwsku/qos.json.j2 ]; then
         sonic-cfggen -j /etc/sonic/init_cfg.json -t /usr/share/sonic/hwsku/qos.json.j2 > /tmp/qos.json
+        qos_cmd="-j /tmp/qos.json"
     fi
 
     sonic-cfggen -p /usr/share/sonic/device/$PLATFORM/$PLATFORM_CONF -k $HWSKU --print-data > /tmp/ports.json
     # change admin_status from up to down; Test cases dependent
     sed -i "s/up/down/g" /tmp/ports.json
-    sonic-cfggen -j /etc/sonic/init_cfg.json -j /tmp/buffers.json -j /tmp/qos.json -j /tmp/ports.json --print-data > /etc/sonic/config_db.json
+    sonic-cfggen -j /etc/sonic/init_cfg.json $buffers_cmd $qos_cmd -j /tmp/ports.json --print-data > /etc/sonic/config_db.json
+fi
+
+if [ "$HWSKU" != "NPU-2P" ]; then
+    sonic-cfggen -t /usr/share/sonic/templates/copp_cfg.j2 > /etc/sonic/copp_cfg.json
```

Comment:
why DPU won't have copp_cfg.json? DPU should also have right?

Reply:
Okay, I have added this for DPU also.